assets/payment/settings_script.js

```diff
@@ -101,6 +101,13 @@ document.addEventListener('DOMContentLoaded', function () {
 			reactivateFunction();
 		});
 	}
+	const prova = document.querySelector('#prova');
+	if (prova) {
+		prova.addEventListener('click', (e) => {
```

Comment:
Codacy found an issue: [The function addEventListener can be unsafe](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5905808)